staging/src/k8s.io/metrics/README.md

```diff
@@ -56,10 +54,9 @@ page](http://kubernetes.io/community/).
 
 You can reach the maintainers of this repository at:
 
-- Slack: #sig-instrumentation (on https://kubernetes.slack.com -- get an
-  invite at slack.kubernetes.io)
-- Mailing List:
-  https://groups.google.com/forum/#!forum/kubernetes-sig-instrumentation
+- Slack: #sig-instrumentation channel at [kubernetes.slack.com](https://kubernetes.slack.com), you can get an
```

Comment:
Direct link to sig-instrumentation channel
https://kubernetes.slack.com/messages/sig-instrumentation